packages/vuetify/src/components/VIcon/VIcon.ts

```diff
@@ -182,7 +176,7 @@ const VIcon = mixins(
     const icon = this.getIcon()
 
     if (typeof icon === 'string') {
-      if (icon.indexOf('svg-') === 0) {
+      if (icon.startsWith('M') && icon.endsWith('Z')) {
```

Comment:
it can end with `Z`, but it can not to do that. I don't know about `M`

Reply:
See https://github.com/vuetifyjs/vuetify/pull/7019#discussion_r276595311

Reply:
:ok_hand:

Reply:
Are there any css icon library that starts the icon name with `M` and ends with `Z`?
We are not accepting `@mdi/js` only. It is supposed to work with any custom svg path.

Reply:
Libraries, apps, icon packs, and others that uses or exports icons and images as svg or path:
https://www.flaticon.com/ (that one has some paths that don't end with `Z` or `z`)
https://icomoon.io/app/
https://simpleicons.org/
https://linearicons.com/free

Please notice that a lot of svg paths ends with `z` instead of `Z`.